components/nimbus/src/tests/helpers.rs

```diff
@@ -405,6 +405,24 @@ pub fn get_single_feature_rollout(slug: &str, feature_id: &str, config: Value) -
     .unwrap()
 }
 
+pub fn get_single_feature_experiment_with(
```

Comment:
Nit: could you document what the `with` is? It's not super clear what the returned experiment is/has.